disnake/ext/commands/base_core.py

```diff
@@ -113,6 +113,7 @@ def __init__(self, func: CommandCallback, *, name: str = None, **kwargs):
         self.qualified_name: str = self.name
         # only an internal feature for now
         self.guild_only: bool = kwargs.get("guild_only", False)
+        self.extras: Dict[str, Any] = kwargs.get("extras") or {}
```

Comment:
```suggestion
        self.extras: Dict[str, Any] = kwargs.get("extras", {})
```
might be better, the prefix command one works similarly

additionally, the docstring of this type was updated in #470, so the new attribute should be added to it as well

Reply:
I used or here intentionally as a couple different places pass `extras` to here with default `None`, to keep consistent with the existing spec. For example, `SubCommandGroup.sub_command` had all its kwargs default to `None`. To follow suit, I did the same, but that means None gets explicitly passed to the underlying implementation here, which is why I had to use or (or some other alternative, but that seemed even lengthier, unless you have a better idea).

As for the new attribute, will do in a bit ^^.

Reply:
I found a issue where the user may pass a dict which is edited *later*. This should be documented as unsupported, with the same disclaimer that `Command.extras` has: `This object may be copied by the library.`

Reply:
Thanks ^^, disclaimers were added as of 0caa4bf5dcef5e374fef37b09e14fcfa4c1efe47.